### PR TITLE
Canonical link, sharing images, Meta namespace for "article:"

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @romanzipp 

as promised here the PR for #26. 

There might be some rough corners and todos. 

Let me know what you think. Happy to reply to any comments/review notes.

Cheers,
Peter

### CODE REVIEW

File: src/Structs/Meta/Article.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        return $this;
+    }
+
+    public function setUnique(bool $unique = true): Struct
+    {
+        $this->unique = $unique;
+
+        return $this;
+    }
```
Comment (romanzipp) [inline src/Structs/Meta/Article.php:29]: I would rather move this method the `romanzipp\Seo\Structs\Struct` class

**Review by romanzipp — CHANGES_REQUESTED** [review]

File: src/Structs/Meta/Article.php (hunk 1 above)
Comment (spekulatius) [reply src/Structs/Meta/Article.php:29]: Sure, no problem 👍️

Comment (spekulatius) [reply src/Structs/Meta/Article.php]: Done @romanzipp

**Comment by romanzipp** [comment]
Nice, looks good! Do you have anything to add or is the PR ready to merge?

**Review by romanzipp — APPROVED** [review]

**Comment by spekulatius** [comment]
Hey @romanzipp 

I can't think of anything really. 

Maybe I stumble upon something or see other chances to improve it further later. For now, this should round it up nicely.

Thank you for all your help so far!

Cheers,
Peter

**Pull request merged**